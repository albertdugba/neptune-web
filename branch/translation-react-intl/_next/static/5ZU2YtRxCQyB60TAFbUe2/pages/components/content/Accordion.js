(window.webpackJsonp=window.webpackJsonp||[]).push([[41],{"65gV":function(n,t,e){var r=e("tsxu"),o=e("dsLa");n.exports=function(n,t){return!t||"object"!==r(t)&&"function"!==typeof t?o(n):t}},CrPR:function(n,t,e){var r=e("e1rg");n.exports=r&&!Symbol.sham&&"symbol"==typeof Symbol.iterator},DZbC:function(n,t,e){var r=e("QKKh"),o=e("QOpd"),i=e("R/wC")("species");n.exports=function(n,t){var e;return o(n)&&("function"!=typeof(e=n.constructor)||e!==Array&&!o(e.prototype)?r(e)&&null===(e=e[i])&&(e=void 0):e=void 0),new(void 0===e?Array:e)(0===t?0:t)}},Hi8P:function(n,t,e){"use strict";var r,o=e("gC0r"),i=e.n(o),c=(e("J/fC"),e("ERkP")),a=e.n(c),u=e("O94r"),s=e.n(u),f=e("/Vl7"),p=e("VehP"),l=e("eKd8"),d=(r={},i()(r,p.a.SMALL,16),i()(r,p.a.MEDIUM,24),r),m=function(n){var t=n.orientation,e=n.size,r=n.disabled,o=n.className,i={className:s()("tw-chevron",{"chevron-color":!r},"".concat([t]).toLowerCase(),o),size:d[e]};return a.a.createElement(f.g,i)};m.Orientation=l.b,m.Size=p.a,m.defaultProps={orientation:m.Orientation.BOTTOM,size:m.Size.SMALL,disabled:!1,className:null},t.a=m},"J/fC":function(n,t,e){},JbFr:function(n,t){n.exports=function(n){if("function"!=typeof n)throw TypeError(String(n)+" is not a function");return n}},O94r:function(n,t,e){var r;!function(){"use strict";var e={}.hasOwnProperty;function o(){for(var n=[],t=0;t<arguments.length;t++){var r=arguments[t];if(r){var i=typeof r;if("string"===i||"number"===i)n.push(r);else if(Array.isArray(r)&&r.length){var c=o.apply(null,r);c&&n.push(c)}else if("object"===i)for(var a in r)e.call(r,a)&&r[a]&&n.push(a)}}return n.join(" ")}n.exports?(o.default=o,n.exports=o):void 0===(r=function(){return o}.apply(t,[]))||(n.exports=r)}()},PjUs:function(n,t,e){var r=e("fyTV"),o=e("xFK5"),i=e("k8sU"),c=e("QI0Q"),a=e("DZbC"),u=[].push,s=function(n){var t=1==n,e=2==n,s=3==n,f=4==n,p=6==n,l=5==n||p;return function(d,m,y,b){for(var O,v,x=i(d),h=o(x),T=r(m,y,3),w=c(h.length),g=0,P=b||a,E=t?P(d,w):e?P(d,0):void 0;w>g;g++)if((l||g in h)&&(v=T(O=h[g],g,x),n))if(t)E[g]=v;else if(v)switch(n){case 3:return!0;case 5:return O;case 6:return g;case 2:u.call(E,O)}else if(f)return!1;return p?-1:s||f?f:E}};n.exports={forEach:s(0),map:s(1),filter:s(2),some:s(3),every:s(4),find:s(5),findIndex:s(6)}},PqPt:function(n,t){function e(t,r){return n.exports=e=Object.setPrototypeOf||function(n,t){return n.__proto__=t,n},e(t,r)}n.exports=e},QOpd:function(n,t,e){var r=e("l9lp");n.exports=Array.isArray||function(n){return"Array"==r(n)}},"R/wC":function(n,t,e){var r=e("4im6"),o=e("lEPA"),i=e("d3yh"),c=e("rdnw"),a=e("e1rg"),u=e("CrPR"),s=o("wks"),f=r.Symbol,p=u?f:f&&f.withoutSetter||c;n.exports=function(n){return i(s,n)||(a&&i(f,n)?s[n]=f[n]:s[n]=p("Symbol."+n)),s[n]}},VehP:function(n,t,e){"use strict";e.d(t,"a",(function(){return r}));var r={EXTRA_SMALL:"xs",SMALL:"sm",MEDIUM:"md",LARGE:"lg",EXTRA_LARGE:"xl"}},VrRy:function(n,t,e){"use strict";e.r(t),e.d(t,"meta",(function(){return k})),e.d(t,"default",(function(){return N}));var r=e("cxan"),o=e("HbGN"),i=e("ERkP"),c=e.n(i),a=e("ZVZ0"),u=e("Qi1R"),s=(e("sL/y"),e("eS6V")),f=e.n(s),p=e("nvWY"),l=e.n(p),d=e("oobO"),m=e.n(d),y=e("65gV"),b=e.n(y),O=e("cVsr"),v=e.n(O),x=e("dsLa"),h=e.n(x),T=e("brTf"),w=e.n(T),g=e("gC0r"),P=e.n(g),E=e("O94r"),S=e.n(E),A=e("v1+v"),C=e("Hi8P"),L=function(n){var t=n.id,e=n.title,r=n.content,o=n.onClick,i=n.isOpen,a=n.index;return c.a.createElement("div",{id:t,className:S()("tw-accordion-item decision p-a-0",{closed:!i})},c.a.createElement("button",{type:"button","aria-expanded":i,onClick:function(){return o(a)},className:"tw-accordion-item btn btn-link p-l-0 text-no-decoration"},c.a.createElement("div",{className:"media p-y-2"},c.a.createElement("div",{className:"media-body text-xs-left"},Object(A.h)(e)?c.a.createElement("span",{className:"h5"},e):e),c.a.createElement("div",{className:"media-right media-middle"},c.a.createElement(C.a,{orientation:i?C.a.Orientation.TOP:C.a.Orientation.BOTTOM})))),c.a.createElement("div",{className:"p-b-3 accordion-content"},r))};L.defaultProps={id:null};var j=L,_=(e("cnZ1"),function(n){function t(n){var e;return l()(this,t),e=b()(this,v()(t).call(this,n)),P()(h()(e),"handleOnClick",(function(n){e.setState((function(t){return{indexOpen:t.indexOpen===n?-1:n}})),e.props.onClick&&e.props.onClick(n)})),e.state={indexOpen:n.indexOpen},e}return w()(t,n),m()(t,[{key:"componentDidUpdate",value:function(n){n.indexOpen!==this.props.indexOpen&&this.setState({indexOpen:this.props.indexOpen})}},{key:"render",value:function(){var n=this;return c.a.createElement(c.a.Fragment,null,this.props.items.map((function(t,e){return c.a.createElement(j,f()({id:t.id,key:t.id||e,index:e,isOpen:e===n.state.indexOpen,onClick:n.handleOnClick},t))})))}}]),t}(i.PureComponent));_.defaultProps={onClick:null,indexOpen:-1};var R=_,M="<Accordion\n  indexOpen={1}\n  items={[\n    {\n      title: 'Item 1',\n      content: 'I can be text',\n    },\n    {\n      title: 'Item 2',\n      content: <span>I can be a node</span>,\n    },\n    {\n      title: 'Item 3',\n      content: 'I can be text',\n    },\n  ]}\n/>;\n",k=(c.a.createElement,{name:"Accordion"}),I={meta:k},V="wrapper";function N(n){var t=n.components,e=Object(o.a)(n,["components"]);return Object(a.a)(V,Object(r.a)({},I,e,{components:t,mdxType:"MDXLayout"}),Object(a.a)(u.b,{code:M,scope:{Accordion:R},mdxType:"LiveEditorBlock"}),Object(a.a)(u.a,{componentName:"Accordion",mdxType:"GeneratePropsTable"}))}N.isMDXComponent=!0},YyzA:function(n,t,e){var r=e("AoMu"),o=e("R/wC"),i=e("zLxc"),c=o("species");n.exports=function(n){return i>=51||!r((function(){var t=[];return(t.constructor={})[c]=function(){return{foo:1}},1!==t[n](Boolean).foo}))}},brTf:function(n,t,e){var r=e("PqPt");n.exports=function(n,t){if("function"!==typeof t&&null!==t)throw new TypeError("Super expression must either be null or a function");n.prototype=Object.create(t&&t.prototype,{constructor:{value:n,writable:!0,configurable:!0}}),t&&r(n,t)}},cVsr:function(n,t){function e(t){return n.exports=e=Object.setPrototypeOf?Object.getPrototypeOf:function(n){return n.__proto__||Object.getPrototypeOf(n)},e(t)}n.exports=e},dsLa:function(n,t){n.exports=function(n){if(void 0===n)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return n}},e1rg:function(n,t,e){var r=e("AoMu");n.exports=!!Object.getOwnPropertySymbols&&!r((function(){return!String(Symbol())}))},eKd8:function(n,t,e){"use strict";e.d(t,"b",(function(){return r})),e.d(t,"a",(function(){return o}));var r={TOP:"top",RIGHT:"right",BOTTOM:"bottom",BOTTOM_RIGHT:"bottom-right",BOTTOM_LEFT:"bottom-left",LEFT:"left",LEFT_TOP:"left-top",RIGHT_TOP:"right-top"},o={TOP_LEFT:"up-left",TOP:"up-center",TOP_RIGHT:"up-right",BOTTOM_LEFT:"down-left",BOTTOM:"down-center",BOTTOM_RIGHT:"down-right"}},eS6V:function(n,t){function e(){return n.exports=e=Object.assign||function(n){for(var t=1;t<arguments.length;t++){var e=arguments[t];for(var r in e)Object.prototype.hasOwnProperty.call(e,r)&&(n[r]=e[r])}return n},e.apply(this,arguments)}n.exports=e},ellU:function(n,t,e){var r=e("OzhJ");n.exports=r("navigator","userAgent")||""},fyTV:function(n,t,e){var r=e("JbFr");n.exports=function(n,t,e){if(r(n),void 0===t)return n;switch(e){case 0:return function(){return n.call(t)};case 1:return function(e){return n.call(t,e)};case 2:return function(e,r){return n.call(t,e,r)};case 3:return function(e,r,o){return n.call(t,e,r,o)}}return function(){return n.apply(t,arguments)}}},gC0r:function(n,t){n.exports=function(n,t,e){return t in n?Object.defineProperty(n,t,{value:e,enumerable:!0,configurable:!0,writable:!0}):n[t]=e,n}},jw1G:function(n,t,e){var r=e("MlVR"),o=e("AoMu"),i=e("d3yh"),c=Object.defineProperty,a={},u=function(n){throw n};n.exports=function(n,t){if(i(a,n))return a[n];t||(t={});var e=[][n],s=!!i(t,"ACCESSORS")&&t.ACCESSORS,f=i(t,0)?t[0]:u,p=i(t,1)?t[1]:void 0;return a[n]=!!e&&!o((function(){if(s&&!r)return!0;var n={length:-1};s?c(n,1,{enumerable:!0,get:u}):n[1]=1,e.call(n,f,p)}))}},nvWY:function(n,t){n.exports=function(n,t){if(!(n instanceof t))throw new TypeError("Cannot call a class as a function")}},oobO:function(n,t){function e(n,t){for(var e=0;e<t.length;e++){var r=t[e];r.enumerable=r.enumerable||!1,r.configurable=!0,"value"in r&&(r.writable=!0),Object.defineProperty(n,r.key,r)}}n.exports=function(n,t,r){return t&&e(n.prototype,t),r&&e(n,r),n}},"sL/y":function(n,t,e){"use strict";var r=e("EtS/"),o=e("PjUs").map,i=e("YyzA"),c=e("jw1G"),a=i("map"),u=c("map");r({target:"Array",proto:!0,forced:!a||!u},{map:function(n){return o(this,n,arguments.length>1?arguments[1]:void 0)}})},t1Iz:function(n,t,e){(window.__NEXT_P=window.__NEXT_P||[]).push(["/components/content/Accordion",function(){return e("VrRy")}])},tsxu:function(n,t){function e(t){return"function"===typeof Symbol&&"symbol"===typeof Symbol.iterator?n.exports=e=function(n){return typeof n}:n.exports=e=function(n){return n&&"function"===typeof Symbol&&n.constructor===Symbol&&n!==Symbol.prototype?"symbol":typeof n},e(t)}n.exports=e},zLxc:function(n,t,e){var r,o,i=e("4im6"),c=e("ellU"),a=i.process,u=a&&a.versions,s=u&&u.v8;s?o=(r=s.split("."))[0]+r[1]:c&&(!(r=c.match(/Edge\/(\d+)/))||r[1]>=74)&&(r=c.match(/Chrome\/(\d+)/))&&(o=r[1]),n.exports=o&&+o}},[["t1Iz",0,1,2,3,7,4,6,11,5]]]);