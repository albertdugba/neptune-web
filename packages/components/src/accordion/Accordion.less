@import (reference) '@transferwise/neptune-css/src/less/addons/_spacing-utilities.less';

@media (--screen-sm-max) {
  .tw-accordion-item {
    .accordion-content {
      .media-body {
        .m-l-0;
      }
    }
  }
}

.tw-accordion-item {
  display: block;
  width: 100%;
  padding: 10px 24px;

  .btn {
    border: none;
  }

  &:hover {
    background-color:
      var(
        --color-background-screen
      ) !important; // overrides bootstrap rule for .decision
  }

  .chevron {
    vertical-align: middle;
  }

  .media {
    color: var(--color-text-primary);
    flex-wrap: wrap;
  }

  .media-body {
    white-space: normal;
  }

  .accordion-content {
    cursor: auto;
  }

  &.closed {
    .accordion-content {
      max-height: 0;
      overflow: hidden;
    }
  }
}
