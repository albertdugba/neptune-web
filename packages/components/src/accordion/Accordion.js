import React, { useState } from 'react';

import PropTypes from 'prop-types';
import AccordionItem from './AccordionItem';
import { Theme } from '../common';
import './Accordion.css';

const Accordion = ({ items, onClick, indexOpen, theme }) => {
  const [itemsOpen, setItemsOpen] = useState(() => items.map((val, index) => index === indexOpen));
  const handleOnClick = (index) => {
    if (onClick) {
      onClick(index);
    }
    const newItems = [...itemsOpen];
    newItems[index] = !itemsOpen[index];
    setItemsOpen(newItems);
  };

  return items.map((item, index) => (
    <AccordionItem
      key={item.id || index}
      onClick={() => handleOnClick(index)}
      open={itemsOpen[index]}
      {...item}
      theme={theme}
    />
  ));
};

Accordion.propTypes = {
  indexOpen: PropTypes.number,
  items: PropTypes.arrayOf(
    PropTypes.shape({
      id: PropTypes.string,
      title: PropTypes.node.isRequired,
      content: PropTypes.node.isRequired,
      icon: PropTypes.node,
    }),
  ).isRequired,
  onClick: PropTypes.func,
  theme: PropTypes.oneOf(['light', 'dark']),
};

Accordion.defaultProps = {
  indexOpen: -1,
  onClick: null,
  theme: Theme.LIGHT,
};

export default Accordion;
