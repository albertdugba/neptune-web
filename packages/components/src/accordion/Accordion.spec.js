import { Freeze, Emoji, New } from '@transferwise/icons';

import { render, screen, userEvent } from '../test-utils';

import Accordion from './Accordion';

describe('Accordion', () => {
  const items = [
    {
      title: 'This is title number one',
      content: 'Lauri Ipsum has been the industry standard dummy text ever since the 1500s.',
      icon: <Freeze />,
    },
    {
      title: 'This is title two',
      content: 'Lauri Ipsum is simply dummy text of the printing and typesetting industry.',
      icon: <New />,
    },
    {
      title: 'This is title three',
      content: 'Lauri Ipsum has been the industry standard dummy text ever since the 1500s.',
      icon: <Emoji />,
    },
  ];

  describe('defaults', () => {
    it('renders an empty list when no items are passed', () => {
      expect(render(<Accordion items={[]} />).container).toMatchSnapshot();
    });
  });

  describe('items', () => {
    it('renders a list with all items closed if items passed', () => {
      render(<Accordion items={items} />);
      const openItem = screen.queryByRole('expanded');
      expect(openItem).not.toBeInTheDocument();
    });

    it('renders a list with a specified item open if indexOpen is set', () => {
      const { container } = render(<Accordion items={items} indexOpen={1} />);
      const openItems = container.querySelectorAll('.tw-accordion-item[aria-expanded="true"]');
      expect(openItems).toHaveLength(1);

      const openItem = container.querySelector(
        '.tw-accordion-item[aria-expanded="true"] .title .title-4',
      );
      expect(openItem.innerHTML).toBe(items[1].title);
    });
  });

  describe('open and close', () => {
    it('opens an item when clicked', () => {
      const { container } = render(<Accordion items={items} />);
      userEvent.click(screen.getAllByRole('button')[0]);

      expect(container).toMatchSnapshot();
      expect(container.querySelectorAll('.closed')).toHaveLength(2);
    });
  });

  describe('onClick', () => {
    it('calls onClick when an item is clicked', () => {
      const onClick = jest.fn();

      render(<Accordion items={items} onClick={onClick} />);

      userEvent.click(screen.getAllByRole('button')[0]);

      expect(onClick).toHaveBeenCalledTimes(1);
      expect(onClick).toHaveBeenCalledWith(0);
    });
  });
});
