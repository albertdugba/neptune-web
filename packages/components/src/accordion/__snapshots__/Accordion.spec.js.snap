// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion defaults renders an empty list when no items are passed 1`] = `<div />`;

exports[`Accordion open and close opens an item when clicked 1`] = `
<div>
  <div
    class="tw-accordion-item p-a-0 tw-accordion-item--light"
  >
    <button
      aria-expanded="true"
      class="tw-accordion-item btn btn-link text-no-decoration decision"
      type="button"
    >
      <div
        class="media"
      >
        <div
          class="hidden-xs hidden-sm media-left p-r-2"
        >
          <span
            aria-hidden="true"
            class="tw-icon tw-icon-freeze "
            role="presentation"
          >
            <svg
              fill="currentColor"
              focusable="false"
              height="24"
              width="24"
            >
              <path
                d="M13.002 5.643L15.6 3.565 14.348 2 12 3.878 9.652 2 8.4 3.565l2.598 2.078v4.621l-4.003-2.31-.5-3.29-1.981.303.452 2.972-2.8 1.094.729 1.867 3.098-1.21L9.996 12l-4.003 2.31-3.098-1.21-.73 1.867 2.801 1.094-.452 2.972 1.98.302.501-3.288 4.003-2.311v4.621L8.4 20.435 9.652 22 12 20.122 14.348 22l1.252-1.565-2.598-2.078v-4.621l4.003 2.31.5 3.29 1.981-.303-.452-2.972 2.8-1.094-.729-1.867-3.098 1.21L14.004 12l4.003-2.31 3.098 1.21.73-1.867-2.801-1.094.452-2.972-1.98-.302-.501 3.288-4.003 2.311V5.643z"
              />
            </svg>
          </span>
        </div>
        <div
          class="media-body title text-xs-left"
        >
          <span
            class="h5 tw-accordion-item__title"
          >
            This is title number one
          </span>
        </div>
        <div
          class="d-flex align-items-center media-right"
        >
          <span
            aria-hidden="true"
            class="tw-icon tw-icon-chevron-up tw-chevron chevron-color top"
            role="presentation"
          >
            <svg
              fill="currentColor"
              focusable="false"
              height="16"
              width="16"
            >
              <path
                clip-rule="evenodd"
                d="M8 4l-6.6 6.653L2.537 11.8 8 6.293l5.463 5.507 1.137-1.147L8 4z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
        </div>
      </div>
    </button>
    <div
      class="accordion-content media"
    >
      <div
        aria-hidden="false"
        class="media-body p-t-2 p-b-3 m-l-5 m-r-5 p-l-2"
      >
        Lauri Ipsum has been the industry standard dummy text ever since the 1500s.
      </div>
    </div>
  </div>
  <div
    class="tw-accordion-item p-a-0 tw-accordion-item--light closed"
  >
    <button
      aria-expanded="false"
      class="tw-accordion-item btn btn-link text-no-decoration decision"
      type="button"
    >
      <div
        class="media"
      >
        <div
          class="hidden-xs hidden-sm media-left p-r-2"
        >
          <span
            aria-hidden="true"
            class="tw-icon tw-icon-new "
            role="presentation"
          >
            <svg
              fill="currentColor"
              focusable="false"
              height="24"
              width="24"
            >
              <path
                d="M17.197 3.966a.518.518 0 01-.231.231l-1.68.84a.517.517 0 000 .926l1.68.84c.1.05.181.131.231.231l.84 1.68c.19.381.735.381.926 0l.84-1.68c.05-.1.131-.181.231-.231l1.68-.84a.517.517 0 000-.926l-1.68-.84a.518.518 0 01-.231-.231l-.84-1.68a.517.517 0 00-.926 0l-.84 1.68z"
              />
              <path
                clip-rule="evenodd"
                d="M10.894 6.553a1 1 0 00-1.788 0L6.92 10.92l-4.368 2.185a1 1 0 000 1.788L6.92 17.08l2.185 4.368a1 1 0 001.788 0l2.185-4.368 4.368-2.185a1 1 0 000-1.788L13.08 10.92l-2.185-4.368zm-2.333 5.56L10 9.237l1.439 2.878a1 1 0 00.447.447L14.764 14l-2.878 1.439a1 1 0 00-.447.447L10 18.764l-1.439-2.878a1 1 0 00-.447-.447L5.236 14l2.878-1.439a1 1 0 00.447-.447z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
        </div>
        <div
          class="media-body title text-xs-left"
        >
          <span
            class="h5 tw-accordion-item__title"
          >
            This is title two
          </span>
        </div>
        <div
          class="d-flex align-items-center media-right"
        >
          <span
            aria-hidden="true"
            class="tw-icon tw-icon-chevron-up tw-chevron chevron-color bottom"
            role="presentation"
          >
            <svg
              fill="currentColor"
              focusable="false"
              height="16"
              width="16"
            >
              <path
                clip-rule="evenodd"
                d="M8 4l-6.6 6.653L2.537 11.8 8 6.293l5.463 5.507 1.137-1.147L8 4z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
        </div>
      </div>
    </button>
    <div
      class="accordion-content media"
    >
      <div
        aria-hidden="true"
        class="media-body p-t-2 p-b-3 m-l-5 m-r-5 p-l-2"
      >
        Lauri Ipsum is simply dummy text of the printing and typesetting industry.
      </div>
    </div>
  </div>
  <div
    class="tw-accordion-item p-a-0 tw-accordion-item--light closed"
  >
    <button
      aria-expanded="false"
      class="tw-accordion-item btn btn-link text-no-decoration decision"
      type="button"
    >
      <div
        class="media"
      >
        <div
          class="hidden-xs hidden-sm media-left p-r-2"
        >
          <span
            aria-hidden="true"
            class="tw-icon tw-icon-emoji "
            role="presentation"
          >
            <svg
              fill="currentColor"
              focusable="false"
              height="24"
              width="24"
            >
              <path
                d="M9.1 11a1.6 1.6 0 100-3.2 1.6 1.6 0 000 3.2zM16.7 9.4a1.6 1.6 0 11-3.2 0 1.6 1.6 0 013.2 0zM16 14a4 4 0 01-8 0h8z"
              />
              <path
                clip-rule="evenodd"
                d="M22 12c0 5.523-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2s10 4.477 10 10zm-2 0a8 8 0 11-16 0 8 8 0 0116 0z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
        </div>
        <div
          class="media-body title text-xs-left"
        >
          <span
            class="h5 tw-accordion-item__title"
          >
            This is title three
          </span>
        </div>
        <div
          class="d-flex align-items-center media-right"
        >
          <span
            aria-hidden="true"
            class="tw-icon tw-icon-chevron-up tw-chevron chevron-color bottom"
            role="presentation"
          >
            <svg
              fill="currentColor"
              focusable="false"
              height="16"
              width="16"
            >
              <path
                clip-rule="evenodd"
                d="M8 4l-6.6 6.653L2.537 11.8 8 6.293l5.463 5.507 1.137-1.147L8 4z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
        </div>
      </div>
    </button>
    <div
      class="accordion-content media"
    >
      <div
        aria-hidden="true"
        class="media-body p-t-2 p-b-3 m-l-5 m-r-5 p-l-2"
      >
        Lauri Ipsum has been the industry standard dummy text ever since the 1500s.
      </div>
    </div>
  </div>
</div>
`;
